mythical-creatures/test/hobbit-test.js

```diff
@@ -39,44 +41,44 @@ describe('Hobbit', function () {
   function timeTravel(num, hobbit) {
     for(var i = 0; i < num; i++) {
       hobbit.celebrateBirthday();
-    };
-  };
+    }
+  }
```

Comment:
@applegrain this is the function I was referencing above.
Pass it a number, then start at 0 and work up to the number - each time adding 1 to the age property until you reach the number that was passed in right? Just want to make sure I'm understanding what this is doing.